biomodels/src/test/java/org/reactome/release/ModelsTSVParserTester.java

```diff
@@ -0,0 +1,65 @@
+package org.reactome.release;
+
+import org.junit.Test;
+
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+public class ModelsTSVParserTester {
+    @Test
+    public void emptyMapFromEmptyFileName() {
+        Map<String, List<String>> pathwayToBioModelsIds =  ModelsTSVParser.parse("");
+        assertTrue("Empty file produces empty pathwayToBioModelsIds map", pathwayToBioModelsIds.isEmpty());
+    }
+
+    @Test
+    public void emptyMapFromUnknownFileName() {
+        Map<String, List<String>> pathwayToBioModelsIds =  ModelsTSVParser.parse("filedoesnotexist.tsv");
+        assertTrue("Unknown file produces empty pathwayToBioModelsIds map", pathwayToBioModelsIds.isEmpty());
+    }
+
+    @Test
+    public void emptyMapFromImproperFile() {
+        Map<String, List<String>> pathwayToBioModelsIds = ModelsTSVParser.parse("src/test/resources/improper_models2pathways.tsv");
```

Comment:
It might be good to set a resource directory class-level constant:
```
private final String RESOURCE_DIR = Paths.get("src", "test", "resources").toFile().getAbsolutePath();
```

You can then do a variable in each test method for the file you need:
```
String filePath = Paths.get(RESOURCE_DIR).toFile().getAbsolutePath();
```